bin/vyper

```diff
@@ -44,16 +64,14 @@ if __name__ == '__main__':
         if args.show_gas_estimates:
             parser_utils.LLLnode.repr_show_gas = True
 
-        if args.format == 'abi':
-            print(compiler.mk_full_signature(code))
-        elif args.format == 'json':
-            print(json.dumps(compiler.mk_full_signature(code)))
-        elif args.format == 'bytecode':
-            print('0x' + compiler.compile(code).hex())
-        elif args.format == 'bytecode_runtime':
-            print('0x' + compiler.compile(code, bytecode_runtime=True).hex())
-        elif args.format == 'ir':
-            print(optimizer.optimize(parse_to_lll(code)))
-        elif args.format == 'asm':
-            lll = optimizer.optimize(parse_to_lll(code))
-            print_asm(compile_lll.compile_to_assembly(lll))
+        if len(args.format) == 1:
+            print(format_output(code, args.format[0]))
+        else:
+            output_dict = {}
```

Comment:
I am not sure we want this as dictionary output here, I think newline deliminated (without a description) is a better option.
Especially because the output becomes inconsistent if you use `json` option.

@fubuloubu what do you think?

Reply:
Newline delimited is probably fine, I think there should be a way to describe what you want output (with ABI, binary, and runtime being the default), and another flag whether you want json or newline delimited pretty print.